client/my-sites/domains/domain-management/edit/navigation/enhanced.jsx

```diff
@@ -246,8 +254,28 @@ class DomainManagementNavigationEnhanced extends React.Component {
 		);
 	}
 
+	getTransferMappedDomain() {
+		const { selectedSite, domain, translate } = this.props;
+
+		const { isEligibleForInboundTransfer } = domain;
+
+		if ( ! isEligibleForInboundTransfer ) {
+			return null;
+		}
+
+		return (
+			<DomainManagementNavigationItem
+				onClick={ this.handleTransferDomainClick }
+				path={ domainTransferIn( selectedSite.slug, domain.name, true ) }
+				materialIcon="sync_alt"
+				text={ translate( 'Transfer your domain to WordPress.com' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Transfer Domain to WordPress.com' )` *ES Score: 7*